src/lib/Scenes/Artwork/Components/__tests__/CommercialButtons-tests.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button } from "@artsy/palette"
+import { mount } from "enzyme"
+import React from "react"
+import { RelayProp } from "react-relay"
+import relay from "react-relay"
+import { CommercialButtons } from "../CommercialButtons"
+
+let artwork = {
+  slug: "abbas-kiarostami-untitled-7",
+  internalID: "5b2b745e9c18db204fc32e11",
+  is_acquireable: false,
+  is_offerable: false,
+  is_inquireable: true,
+}
+
+describe("CommercialButtons", () => {
+  artwork = {
+    slug: "abbas-kiarostami-untitled-7",
+    internalID: "5b2b745e9c18db204fc32e11",
+    is_acquireable: false,
+    is_offerable: false,
+    is_inquireable: true,
+  }
+  it("renders button for Contact Gallery button if is_inquireable", () => {
+    const component = mount(<CommercialButtons relay={{ environment: {} } as RelayProp} artwork={artwork as any} />)
+    expect(component.text()).toContain("Contact gallery")
+  })
+
+  it("renders Make Offer button if is_offerable", () => {
+    artwork = {
+      slug: "abbas-kiarostami-untitled-7",
+      internalID: "5b2b745e9c18db204fc32e11",
+      is_acquireable: false,
+      is_offerable: false,
+      is_inquireable: true,
+    }
+    const component = mount(<CommercialButtons relay={{ environment: {} } as RelayProp} artwork={artwork as any} />)
+    expect(component.text()).toContain("Contact gallery")
+  })
+
+  it("renders Buy Now button if is_acquireable", () => {
+    artwork = {
+      slug: "abbas-kiarostami-untitled-7",
+      internalID: "5b2b745e9c18db204fc32e11",
+      is_acquireable: true,
+      is_offerable: false,
+      is_inquireable: false,
+    }
+    const component = mount(<CommercialButtons relay={{ environment: {} } as RelayProp} artwork={artwork as any} />)
+    expect(component.text()).toContain("Buy now")
+  })
+
+  it("renders both Buy Now and Make Offer buttons when is_offerable and is_acquireable", () => {
+    artwork = {
+      slug: "abbas-kiarostami-untitled-7",
+      internalID: "5b2b745e9c18db204fc32e11",
+      is_acquireable: true,
+      is_offerable: true,
+      is_inquireable: false,
+    }
+    const component = mount(<CommercialButtons relay={{ environment: {} } as RelayProp} artwork={artwork as any} />)
+    expect(
+      component
+        .find(Button)
+        .at(0)
+        .text()
+    ).toContain("Buy now")
+    expect(
+      component
+        .find(Button)
+        .at(1)
+        .text()
+    ).toContain("Make offer")
+  })
+
+  it("commits the Buy Now mutation", () => {
+    artwork = {
+      slug: "abbas-kiarostami-untitled-7",
+      internalID: "5b2b745e9c18db204fc32e11",
+      is_acquireable: true,
+      is_offerable: true,
+      is_inquireable: false,
+    }
+
+    const component = mount(<CommercialButtons relay={{ environment: {} } as RelayProp} artwork={artwork as any} />)
+    const BuyNowButton = component.find(Button).at(0)
+
+    relay.commitMutation = jest.fn()
+    BuyNowButton.props().onPress()
+
+    expect(relay.commitMutation).toHaveBeenCalled()
```

Comment:
Instead of verifying that we call `commitMutation`, can we instead mock the mutation response directly and verify that after clicking the button/receiving that response, we call `Switchboard` with the correct value?